pyintegration/tests/test_table.py

```diff
@@ -139,10 +170,8 @@ def test_where_in(self):
             self.assertLessEqual(unique_table.size, result_table.size)
 
         with self.subTest("where-not-in filter"):
-            result_table2 = self.test_table.where_not_in(unique_table, cols=["c"])
-            self.assertEqual(
-                result_table.size, self.test_table.size - result_table2.size
-            )
```

Comment:
Did you mean to delete this one?

Reply:
yes, the single value case covers both the original test and this change